### PR TITLE
Update scss_lint gem version

### PR DESCRIPTION
**Pull request by tanukiti1987**
Now [scss-lint](https://github.com/brigade/scss-lint) latest version is `0.43.2`.

So let's update `pronto-scss` together!

I fixed the parameter for `SCSSLint::Runner#run`.

And I would like to use this patch on my app, as next `pronto-scss` version :)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,8 @@
+before_install:
+  - gem update bundler
```
Comment (mmozuras) [inline .travis.yml:2]: Why is this change needed?

**Comment by mmozuras** [comment]
Thanks! Looks good, would just like a comment on `.travis.yml` changes :smile:

File: .travis.yml (hunk 1 above)
Comment (tanukiti1987) [inline .travis.yml:2]: I faced the problem on build failure at travis CI. (ref. https://travis-ci.org/mmozuras/pronto-scss/builds/101080452 https://s3.amazonaws.com/archive.travis-ci.org/jobs/101080453/log.txt)

This problem occurred because the bundler is old. (ref. http://stackoverflow.com/questions/34363675/refinery-nomethoderror-undefined-method-spec-for-nilnilclass )
So when before building, we have to update bundler :+1:

**Comment by mmozuras** [comment]
@tanukiti1987 cool, thanks :bow:

**Comment by mmozuras** [comment]
Released as `0.5.1`

**Comment by tanukiti1987** [comment]
super!
